### PR TITLE
[entity3268] UI Updates

### PR DESCRIPTION
**Pull request by jeznorth**
*Issue #:*
https://github.com/bcgov/entity/issues/entity3268

UI Updates / Clean-up

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/OrgAdminContact.vue [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+    <div  v-for="(member, index) in getActiveAdmins()" v-bind:key="index + 1">
+      <div v-if="!anonAccount">
+        <div>
+          {{ member.user.firstname }} {{ member.user.lastname }}
+        </div>
+        <div v-if="member.user.contacts[0].email">
+          {{ member.user.contacts[0].email }}
+        </div>
+        <div v-if="member.user.contacts[0].phone">
+          {{ member.user.contacts[0].phone }}
```
Comment (nitheesh-aot) [inline auth-web/src/components/auth/OrgAdminContact.vue:12]: This should have the null check, right? incase if user.contacts is null or undefined?

@saravanpa-aot

File: auth-web/src/components/auth/AccountLoginOptionPicker.vue [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
         flat
-        outlined
         hover
-        elevation="1"
-        class="account-card text-center pa-7 d-flex flex-column"
         @click="selectAuthType(authOption.type)"
-        :class="{ 'active': authType === authOption.type }"
       >
         <div class="account-type__icon mb-6 mt-2">
           <v-icon color="grey">{{authOption.icon}}</v-icon>
```
Comment (nitheesh-aot) [inline auth-web/src/components/auth/AccountLoginOptionPicker.vue:18]: The BCeID logo will be missing here.
I have used the 2FA icon as a substitute, are we planning to keeping that?

File: auth-web/src/components/auth/OrgAdminContact.vue (hunk 1 above)
Comment (saravanpa-aot) [reply auth-web/src/components/auth/OrgAdminContact.vue:12]: @sumesh-aot

File: auth-web/src/components/auth/AccountLoginOptionPicker.vue (hunk 2 above)
Comment (jeznorth) [reply auth-web/src/components/auth/AccountLoginOptionPicker.vue:18]: Fine for now - I'll get the BCeID asset in there in my next PR

File: auth-web/src/components/auth/OrgAdminContact.vue (hunk 1 above)
Comment (sumesh-aot) [reply auth-web/src/components/auth/OrgAdminContact.vue:12]: Yes, I will fix it in my next PR

**Pull request merged**